packages/tools/pie-icons-react/package.json

```diff
@@ -34,7 +34,7 @@
     "build:lib": "rollup -c",
     "build": "yarn clean && yarn build:icons && yarn build:lib",
     "clean": "run -T rimraf -rf ./icons/",
-    "prepublishOnly": "yarn build"
+    "test": "run jest"
```

Comment:
Do we know why we had it or if we really don't need this "prepublishOnly" script anymore?

I ask because it is one of the known [npm lifecycle scripts](https://docs.npmjs.com/cli/v9/using-npm/scripts#:~:text=for%20more%20info.-,prepublishOnly,-Runs%20BEFORE%20the)

Reply:
thanks for spotting! We do need this but I removed it because it made it easier to yalc and test my changes locally. I meant to add it back in - turns out I did but forgot to push up the change 🤦